include/nbl/video/IGPURayTracingPipeline.h

```diff
@@ -61,8 +81,9 @@ class IGPURayTracingPipeline : public IBackendObject, public asset::IRayTracingP
                 return retval;
             }
 
-            inline std::span<const asset::IPipelineBase::SShaderSpecInfo> getShaders() const { return shaders; }
+            inline std::span<const spec_info_t> getShaders() const { return shaders; }
```

Comment:
the creation params shouldn't need a getter, you need 6 different members (5 spans and 1 raygen)